lib/utils/tests/age.test.ts

```diff
@@ -0,0 +1,25 @@
+import { isUnderEighteen } from "../age";
+
+beforeAll(() => {
+  jest.useFakeTimers("modern");
```

Comment:
Why do you need fake timers here?

Reply:
This is needed to mock the Date constructor.